dataforest/core/BranchSpec.py

```diff
@@ -17,37 +17,37 @@ class Spec(list):
         spec: core representation which this class wraps
     Examples:
         >>> # NOTE: conceptual illustration only, not real processes
-        >>> spec = [
+        >>> branch_spec = [
         >>>     {
-        >>>         "process": "normalize",
-        >>>         "params": {
+        >>>         "_PROCESS_": "normalize",
+        >>>         "_PARAMS_": {
         >>>             "min_genes": 5,
         >>>             "max_genes": 5000,
         >>>             "min_cells": 5,
         >>>             "nfeatures": 30,
         >>>             "perc_mito_cutoff": 20,
         >>>             "method": "seurat_default",
         >>>         }
-        >>>         "subset": {
+        >>>         "_SUBSET_": {
         >>>             "indication": {"disease_1", "disease_3"},
         >>>             "collection_center": "mass_general",
         >>>         },
-        >>>         "filter": {
+        >>>         "_FILTER_": {
         >>>             "donor": "D115"
         >>>         }
         >>>     },
         >>>     {
-        >>>         "process": "reduce",    # dimensionality reduction
-        >>>         "alias": "linear_dim_reduce",
-        >>>         "params": {
+        >>>         "_PROCESS_": "reduce",    # dimensionality reduction
+        >>>         "_ALIAS_": "linear_dim_reduce",
+        >>>         "_PARAMS_": {
         >>>             "algorithm": "pca",
         >>>             "n_pcs": 30,
         >>>         }
         >>>     },
         >>>     {
-        >>>         "process": "reduce",
-        >>>         "alias": "nonlinear_dim_reduce",
-        >>>         "params": {
+        >>>         "_PROCESS_": "reduce",
+        >>>         "_ALIAS_": "nonlinear_dim_reduce",
+        >>>         "_PARAMS_": {
         >>>             "algorithm": "umap",
```

Comment:
Yeah, that's a good point. I do want to demonstrate how aliases work, and since this is in dataforest, I figured it didn't necessarily have to be accurate to cellforest. But maybe I should make it more distinct from cellforest so people don't get confused